tests/importance_tests/test_mean_decrease_impurity.py

```diff
@@ -31,10 +32,10 @@ def test_mean_decrease_impurity_importance_evaluator_n_trees() -> None:
     study.optimize(objective, n_trials=3)
 
     evaluator = MeanDecreaseImpurityImportanceEvaluator(n_trees=10, seed=0)
-    param_importance = evaluator.evaluate(study)
```

Comment:
In this test file, I think calling `evaluator.evalute(study)` directly is good approach to focus the behavior of `MeanDecreaseImpurityImportanceEvaluator`. What do you think?

Reply:
Changes in `tests/integration_tests/test_shap.py` and `tests/importance_tests/test_fanova.py` are also.

Reply:
Hmm, I don't think we should test interfaces that are deemed private (not to be called directly by user) in the document.
But if you consider them public, then we should test that also I think.

Reply:
Maybe we should also discuss that in #3589.

Reply:
> Hmm, I don't think we should test interfaces that are deemed private (not to be called directly by user) in the document. But if you consider them public, then we should test that also I think.

Hm, I don't think that. Given that logic, we cannot write unittests for storage as well. It doesn't important that the interface is public to end-users.